app/src/main/java/com/example/trashminder/presentation/createdReminder/CustomItem.kt

```diff
@@ -93,37 +95,24 @@ private fun CardItem(
             ) {
                 Column(modifier = Modifier.padding(10.dp)) {
                     Text(
-                        text = type,
+                        text = reminder.type.toLocalizedString(context),
```

Comment:
A better way of handling this is using `stringResource`. Apparently this function is available in every @Composable function and you can use it to get a string. This is based on the assumption that a composable always has a context since it is a view so it is safe to fetch the string. This would then become 
`stringResource(reminder.type.toResourceId())`. Where `toResourceId` is the old `toLocalizedString(context)`